api/controllers/v0.3/IlsClient.js

```diff
@@ -297,34 +299,50 @@ IlsClient.PATRON_AGENCY_FIELD_TAG = "158";
 IlsClient.NOTICE_PREF_FIELD_TAG = "268";
 IlsClient.NOTE_FIELD_TAG = "x";
 // Standard and temporary expiration times
-IlsClient.STANDARD_EXPIRATION_TIME = 1095; // days
+IlsClient.STANDARD_EXPIRATION_TIME = 1095; // days, 3 years
 IlsClient.TEMPORARY_EXPIRATION_TIME = 30; // days
 IlsClient.WEB_APPLICANT_EXPIRATION_TIME = 90; // days
 // Ptypes for various library card offerings
 IlsClient.WEB_APPLICANT_PTYPE = 1;
-IlsClient.NO_PRINT_ADULT_METRO_PTYPE = 2;
-IlsClient.NO_PRINT_ADULT_NYS_PTYPE = 3;
+IlsClient.SIMPLYE_METRO_PTYPE = 2;
+IlsClient.SIMPLYE_NON_METRO_PTYPE = 3;
 IlsClient.ADULT_METRO_PTYPE = 10;
 IlsClient.ADULT_NYS_PTYPE = 11;
 IlsClient.SENIOR_METRO_PTYPE = 20;
 IlsClient.SENIOR_NYS_PTYPE = 21;
+// The following two p-types don't have a code yet.
+// Using 101 for now but MUST be updated.
+IlsClient.DISABLED_METRO_NY_PTYPE = 101;
+IlsClient.HOMEBOUND_NYC_PTYPE = 101;
 IlsClient.TEEN_METRO_PTYPE = 50;
 IlsClient.TEEN_NYS_PTYPE = 51;
 IlsClient.REJECTED_PTYPE = 101;
 IlsClient.ILS_ERROR = "-1";
 IlsClient.PTYPE_TO_TEXT = {
   WEB_APPLICANT_PTYPE: "Web applicant (No Borrowing)",
-  NO_PRINT_ADULT_METRO_PTYPE: "Adult 19-64 Metro (3 Year, No Print Borrowing)",
-  NO_PRINT_ADULT_NYS_PTYPE: "Adult 19-64 NY State (3 Year, No Print Borrowing)",
-  ADULT_METRO_PTYPE: "Adult 19-64 Metro (3 Year)",
-  ADULT_NYS_PTYPE: "Adult 19-64 NY State (3 Year)",
+  ADULT_METRO_PTYPE: "Adult 18-64 Metro (3 Year)",
+  ADULT_NYS_PTYPE: "Adult 18-64 NY State (3 Year)",
   SENIOR_METRO_PTYPE: "Senior, 65+, Metro (3 Year)",
   SENIOR_NYS_PTYPE: "Senior, 65+, NY State (3 Year)",
+  DISABLED_METRO_NY_PTYPE: "Disabled Metro NY (3 Year)",
+  HOMEBOUND_NYC_PTYPE: "Homebound NYC (3 Year)",
+  SIMPLYE_METRO_PTYPE: "SimplyE Metro",
+  SIMPLYE_NON_METRO_PTYPE: "SimplyE Non-Metro",
   TEEN_METRO_PTYPE: "Teen Metro (3 Year)",
   TEEN_NYS_PTYPE: "Teen NY State (3 Year)",
   REJECTED_PTYPE: "Rejected",
   ILS_ERROR: "Unable to create in ILS",
 };
+IlsClient.CAN_CREATE_DEPENDENTS = [
```

Comment:
@killerpuppytails here's the list of the eight valid p-types that can create dependents. I set a temporary `101` p-type for the "Disabled Metro NY (3 Year)" and "Homebound NYC (3 Year)" patron types until we get the right code.

Reply:
Thanks for tagging me @EdwinGuzman . This was very useful to look through so I ensure there's a path to accommodate teen parents, even while we're not supporting their p-type.